src/settings-ui/Settings.UI.Library/AlwaysOnTopProperties.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.PowerToys.Settings.UI.Library
+{
+    public class AlwaysOnTopProperties
+    {
+        public static readonly HotkeySettings DefaultHotkeyValue = new HotkeySettings(true, true, false, true, 0x54);
```

Comment:
This differs from AlwaysOnTopSettings class.

Reply:
Thanks for the catch!